controllers/participation/aggregator/main.go

```diff
@@ -4,35 +4,31 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	"os"
-
+	"github.com/CollActionteam/collaction_backend/internal/constants"
```

Comment:
Sorry, I didn't understand. 
I use Intellij to code that And I'm looking at the code right now and looks like sorted to me. Can you tell me what's not sorted, please?

Reply:
Yes, there is a folder inside the AWS folder (emailContact), because of this, I have created one folder for participation controllers. I can change the folder without problem

I didn't create an HTTP controller because this project is a lambda project or we will create a web server too?

Reply:
in the meeting the team decided (not sure), implementing the HTTP handler is not required yet.